### DISCUSSION TITLE
How to use an existing joomla website / database?

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomlatools/joomlatools-pages-server#97] [Q&A] [answered]
Hi @johanjanssens,

Is it possible to use joomlatools server with existing joomla website / database?

**Comment by johanjanssens** [comment, marked as answer]
Yes, thats not that hard, here are the steps:

1. Create the `sites_foo` database with phpmyadmin (http://phpmyadmin.localhost) 
2. Import the data with phpmyadmin 
3. Put the files  in the sites folder, create a subfolder for the site
4. edit configuration.php for mysql 

```
public $host = 'localhost';              
public $user = 'admin';                       
public $password = 'admin';                  
public $db = 'sites_foo';     
```

**Reply by alexandreelise** [reply]
As I would expect. Thanks for clarifying.
